<!DOCTYPE html>
<html><head><title>KeyVaultResponseError</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">3 references to KeyVaultResponseError</div><div class="rA">Microsoft.Azure.Services.AppAuthentication (1)</div><div class="rG" id="Microsoft.Azure.Services.AppAuthentication"><div class="rF"><div class="rN">Clients\KeyVault\KeyVaultClient.cs (1)</div>
<a href="../Clients/KeyVault/KeyVaultClient.cs.html#76"><b>76</b>throw new Exception($&quot;{<i>KeyVaultResponseError</i>} KeyVault ResponseCode: {response.StatusCode}, Message: {exceptionText}&quot;);</a>
</div>
</div>
<div class="rA">Microsoft.Azure.Services.AppAuthentication.Unit.Tests (2)</div><div class="rG" id="Microsoft.Azure.Services.AppAuthentication.Unit.Tests"><div class="rF"><div class="rN">ClientCertificateAccessTokenProviderTests.cs (1)</div>
<a href="../../Microsoft.Azure.Services.AppAuthentication.Unit.Tests/ClientCertificateAccessTokenProviderTests.cs.html#293"><b>293</b>Assert.Contains(KeyVaultClient.<i>KeyVaultResponseError</i>, exception.Message);</a>
</div>
<div class="rF"><div class="rN">KeyVaultClientTests.cs (1)</div>
<a href="../../Microsoft.Azure.Services.AppAuthentication.Unit.Tests/KeyVaultClientTests.cs.html#124"><b>124</b>Assert.Contains(KeyVaultClient.<i>KeyVaultResponseError</i>, exception.Message);</a>
</div>
</div>
</body></html>